source/2020-05-01-the-ember-times-issue-146.md

```diff
@@ -9,19 +9,31 @@ responsive: true
 <SAYING-HELLO-IN-YOUR-FAVORITE-LANGUAGE> Emberistas! 🐹
 
 <SOME-INTRO-HERE-TO-KEEP-THEM-SUBSCRIBERS-READING>
-
+Yehuda's Octane Tutorial continued 🔥,
+  
 READMORE
 
 ---
 
-## [Section title in sentence case 🐹](section-url)
+## [Yehuda's Octane tutorial continued 🔥](https://yehudakatz.com/2020/03/25/ember-octane-lets-go/)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+Our very own [Yehuda Katz (@wycats)](https://github.com/wycats) wrote a blog series on [Octane tutorial](https://yehudakatz.com/2020/03/25/ember-octane-lets-go/), and there are two new posts added to it.
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+The third part of the tutorial, [Pulling Out Data](https://yehudakatz.com/2020/03/30/ember-octane-a-data-file/), talks about moving data from HTML into a separate file. This is the part in the series that introduces you to writing Javascript for the first time. You'll also learn how to build your helpers and writing unit tests for them.
```

Comment:
I think it'd be

```suggestion
The third part of the tutorial, [Pulling Out Data](https://yehudakatz.com/2020/03/30/ember-octane-a-data-file/), talks about moving data from HTML into a separate file. This is the part in the series that introduces you to writing Javascript for the first time. You'll also learn how to build your helpers and write unit tests for them.
```

What do you think?

Reply:
Yeah! What you have suggested sounds perfect!